source/Private/Test-TokenExpired.ps1

```diff
@@ -26,15 +24,13 @@ function Test-TokenExpired {
     [CmdletBinding()]
     param ()
 
-    Confirm-FabricAuthToken | Out-Null
-
-    Write-Message -Message "Validating token..." -Level Debug
+    Write-Message -Message "Validating token..." -Level Verbose
 
     try {
         # Ensure required properties have valid values
-        if ([string]::IsNullOrWhiteSpace($FabricConfig.TenantIdGlobal) -or
+        if ([string]::IsNullOrWhiteSpace($FabricConfig.TenantId) -or
             [string]::IsNullOrWhiteSpace($FabricConfig.TokenExpiresOn)) {
```

Comment:
Instead of creating an error and red text unnecessarily. Could we call Connect-FabricAccount here instead?